site/content/overview/about.md

```diff
@@ -0,0 +1,36 @@
+# About the AWS Deploy Tool
+
+** AWS Deploy Tool** is an interractive tooling for the .NET CLI and the AWS Toolkit for Visual Studio that helps deploy .NET applications with minimum AWS knowledge, and with the fewest clicks or commands. It works by analyzing .NET codebases and guiding developers to the right AWS service. It then selects the right deployment service, builds and packages your application, generates the right IAM roles, and creates the deployment infrastructure. It allows for a quick and easy Proof of concept (POC), smooth graduation to CI/CD, and a gradual ramp up on AWS knowledge.
+
+  > Note: AWS Deploy Tool is currently in Developer Preview.
+
+## Key capabilities
+
+AWS Deploy Tool has the following capabilities:
+
+* **Compute recommendations for your application** – Get recommendations about the type of compute best suited for your application based on the application type.
+* **Dockerfile  generation** - The tool will generate the Dockerfile .... if needed, otherwise an existing Dockerfile will be used.
+* **Auto packaging and deployment** – Auto-generate IAM roles and permissions your application requires, build the deployment artifacts, generate a deployment CDK project, provision the infrastructure and  deploy your application to the chosen AWS compute.
```

Comment:
Remove "Auto-generate IAM roles and permissions your application requires"

Reply:
done